### PR TITLE
Update all templates' dd version, readme, added example env/config

### PR DESCRIPTION
**Pull request by AwesomeStickz**
This PR updates dd version of all templates to v16, updates readme in templates where the updates are needed, and added example env/config file for templates where they're missing

### CODE REVIEW

File: template/nodejs/config.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "token": ""
+}
```
Comment (lts20050703) [inline template/nodejs/config.json:3]: @AwesomeStickz Consider using .env instead of config.json?

Comment (AwesomeStickz) [reply template/nodejs/config.json]: Uses .env now

File: template/beginner/README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 This template is designed for the beginner developer to start coding discord bots.
 
+It uses v16 of discordeno, make sure to install the latest version when you use it.
```
Comment (Skillz4Killz) [inline template/beginner/README.md:5]: I don't like this idea of a specific version number.
```suggestion
Make sure to install the latest version when you use it.
```

File: template/beginner/deps.ts [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+export * from "https://deno.land/x/discordeno@16.0.1/mod.ts";
+export * from "https://deno.land/x/discordeno@16.0.1/plugins/mod.ts";
 
 // Terminal Colors!
 export * from "https://deno.land/std@0.117.0/fmt/colors.ts";
 // Get data from .env files
 export { config as dotEnvConfig } from "https://deno.land/x/dotenv@v3.1.0/mod.ts";
 // Database, thx Tri!
-export * from "https://deno.land/x/kwik@1.3.1/mod.ts";
+export { decode as KwikDecode, encode as KwikEncode, Kwik } from "https://deno.land/x/kwik@v1.2.1/mod.ts";
```
Comment (Skillz4Killz) [inline template/beginner/deps.ts:9]: Why was this downgraded?

File: template/bigbot/.env.example [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# For Prisma use
```
Comment (Skillz4Killz) [inline template/bigbot/.env.example:1]:
```suggestion
# For Prisma use. Remember to remove the `[` and `]`
```

File: template/bigbot/README.md [hunk 5]
```diff
@@ -6,6 +6,8 @@ This template is designed for bots that aim or are already in millions of Discor
 currently Deno & Bun are not ready to run something at such a scale. The general idea of this template can be modified
 for any other runtime if this improves in the future.
 
+This template uses v16 of discordeno, make sure to install the latest version when you use it.
```
Comment (Skillz4Killz) [inline template/bigbot/README.md:9]:
```suggestion
Make sure to install the latest version when you use it.
```

File: template/minimal/README.md [hunk 6]
```diff
@@ -2,6 +2,12 @@
 
 Just the minimum to get a working bot using interactions.
 
+This template uses v16 of discordeno, make sure to install the latest version when you use it.
```
Comment (Skillz4Killz) [inline template/minimal/README.md:5]:
```suggestion
Make sure to install the latest version when you use it.
```

File: template/beginner/deps.ts (hunk 3 above)
Comment (AwesomeStickz) [reply template/beginner/deps.ts:9]: That should be by mistake, will fix

**Pull request merged**